types/hellosign-embedded/index.d.ts

```diff
@@ -1,272 +1,213 @@
+
+        static events: Events;
+
+        static locales: Locales;
+
+        static messages: Messages;
+
+        static version: string;
+
+        readonly element: HTMLElement;
+
+        readonly isOpen: boolean;
+
+        readonly isReady: boolean;
+
+        open: (url: string, opts?: Options) => void;
+
+        close: () => void;
+
+        on(name: Events['CLOSE'], cb: () => void): void;
+        on(name: Events['CANCEL'], cb: () => void): void;
+        on(name: Events['CREATE_TEMPLATE'], cb: (data: CreateTemplateEventPyload) => void): void;
+        on(name: Events['DECLINE'], cb: (data: DeclineEventPyload) => void): void;
+        on(name: Events['ERROR'], cb: (data: ErrorEventPyload) => void): void;
+        on(name: Events['FINISH'], cb: () => void): void;
+        on(name: Events['MESSAGE'], cb: (data: MessageEventPayload) => void): void;
+        on(name: Events['OPEN'], cb: (data: OpenEventPayload) => void): void;
+        on(name: Events['READY'], cb: (data: ReadyEventPayload) => void): void;
+        on(name: Events['REASSIGN'], cb: (data: ReassignEventPayload) => void): void;
+        on(name: Events['SEND'], cb: (data: SendEventPayload) => void): void;
+        on(name: Events['SIGN'], cb: (data: SignEventPayload) => void): void;
+
+        once(name: Events['CLOSE'], cb: () => void): void;
+        once(name: Events['CANCEL'], cb: () => void): void;
+        once(name: Events['CREATE_TEMPLATE'], cb: (data: CreateTemplateEventPyload) => void): void;
+        once(name: Events['DECLINE'], cb: (data: DeclineEventPyload) => void): void;
+        once(name: Events['ERROR'], cb: (data: ErrorEventPyload) => void): void;
+        once(name: Events['FINISH'], cb: () => void): void;
+        once(name: Events['MESSAGE'], cb: (data: MessageEventPayload) => void): void;
+        once(name: Events['OPEN'], cb: (data: OpenEventPayload) => void): void;
+        once(name: Events['READY'], cb: (data: ReadyEventPayload) => void): void;
+        once(name: Events['REASSIGN'], cb: (data: ReassignEventPayload) => void): void;
+        once(name: Events['SEND'], cb: (data: SendEventPayload) => void): void;
+        once(name: Events['SIGN'], cb: (data: SignEventPayload) => void): void;
+
+        off(name: Events['CLOSE'], cb?: CB): void;
+        off(name: Events['CANCEL'], cb?: CB): void;
+        off(name: Events['CREATE_TEMPLATE'], cb?: CB): void;
+        off(name: Events['DECLINE'], cb?: CB): void;
+        off(name: Events['ERROR'], cb?: CB): void;
+        off(name: Events['FINISH'], cb?: CB): void;
+        off(name: Events['MESSAGE'], cb?: CB): void;
+        off(name: Events['OPEN'], cb?: CB): void;
+        off(name: Events['READY'], cb?: CB): void;
+        off(name: Events['REASSIGN'], cb?: CB): void;
+        off(name: Events['SEND'], cb?: CB): void;
+        off(name: Events['SIGN'], cb?: CB): void;
     }
 }
 
-declare var HelloSign: HelloSign.HelloSignStatic;
-
-export as namespace HelloSign;
```

Comment:
`hellosign-embedded` is a UMD module and declares a global value `HelloSign` when used as a script rather than a module: https://www.unpkg.com/browse/hellosign-embedded@2.8.0/umd/embedded.development.js. The `export as namespace HelloSign;` declaration indicates that the module exports may also be available as a global with the provided name. I would recommend restoring this line to maintain that capability.